packages/iris-grid/src/DateUtils.ts

```diff
@@ -324,11 +346,11 @@ class DateUtils {
    * In unit test, DateWrapper is just a number provided in millis, so handles that case.
    * @param {DateWrapper|number} dateWrapper The DateWrapper object, or time in millis
    */
-  static getJsDate(dateWrapper) {
-    if (dateWrapper.asDate) {
-      return dateWrapper.asDate();
+  static getJsDate(dateWrapper: DateWrapper | number): Date {
+    if (typeof dateWrapper === 'number') {
```

Comment:
`DateWrapper` is part of the JS API data types, so no predicate for it. Reversed so I didn't need to create one. Could go here since it is a `DateUtil`, or could make a `jsapi-utils` package that's just TS stuff?

Reply:
Okay keep it reversed. Yeaaaa it should probably go in the shim or in utils... push that off for later.